README.md

```diff
@@ -1,33 +1,33 @@
 # AnimatedTextInput
-iOS custom text input component used in Jobandtalent app.
+iOS custom text input component used in the [Jobandtalent app](https://itunes.apple.com/app/id665060895).
 
-![](https://github.com/jobandtalent/AnimatedTextInput/blob/master/Src/general.gif)
+![](https://github.com/jobandtalent/AnimatedTextInput/blob/master/Assets/general.gif)
 
 ## Installation
 Use cocoapods to install this custom control in your project.
 
 ```
-pod 'AnimatedTextInput'
+pod 'AnimatedTextInput', '~> 0.1.0'
```

Comment:
It's a good point. TBH, I double checked some of the most famous iOS OSS projects and most of them use the "optimistic operator" approach, but I agree it can get out of sync quickly...